config/config.go

```diff
@@ -112,6 +112,15 @@ func FromFlags() (*Config, error) {
 		return nil, fmt.Errorf("EVM network ID not supported")
 	}
 
+	switch flowChainID {
+	case "flow-previewnet":
+		cfg.FlowNetworkID = flowGo.Previewnet
+	case "flow-emulator":
+		cfg.FlowNetworkID = flowGo.Emulator
+	default:
+		return nil, fmt.Errorf("flow network ID not supported, only possible to specify 'flow-previewnet' or 'flow-emulator'")
```

Comment:
Should we proactively add the `testnet` & `mainnet`? Or is it too early to do so? Since they don't yet have EVM support.

Reply:
I would say too early